pkg/kubelet/kuberuntime/logs/logs.go

```diff
@@ -354,10 +355,15 @@ func ReadLogs(ctx context.Context, path, containerID string, opts *LogOptions, r
 				if err != nil {
 					return err
 				}
+				if tryrecreated {
+					tryrecreated = false
+					recreated = true
+				}
 				if recreated {
 					newF, err := os.Open(path)
 					if err != nil {
 						if os.IsNotExist(err) {
+							tryrecreated = true
```

Comment:
Can you add some comments for this line?
// If the new log file has not yet been created, try again to read

Reply:
I added comments to the source code as you suggested.